importer/src/map_config.rs

```diff
@@ -0,0 +1,74 @@
+use abstio::{CityName, MapName};
+use geom::Distance;
+use map_model::DrivingSide;
+
+/// Given the name of a map, configure its import.
+///
+/// Note this was once expressed as config files for every city. That was less maintainable; most
+/// places used default values that were copied around.
+pub fn config_for_map(name: &MapName) -> convert_osm::Options {
+    // Some maps have extra procedurally generated houses. Just see if a file in a canonical
+    // location exists.
+    let procgen_houses = name.city.input_path("procgen_houses.json");
+    let extra_buildings = if abstio::file_exists(&procgen_houses) {
+        Some(procgen_houses)
+    } else {
+        None
+    };
+
+    convert_osm::Options {
+        map_config: map_model::MapConfig {
+            driving_side: match name.city.country.as_ref() {
+                "gb" | "jp" | "nz" | "sg" => DrivingSide::Left,
+                _ => DrivingSide::Right,
+            },
+            bikes_can_use_bus_lanes: name.city.country != "pl",
+            inferred_sidewalks: name.city.country != "pl",
+            street_parking_spot_length: if name.city == CityName::new("ca", "montreal") {
+                Distance::meters(6.5)
+            } else {
+                Distance::meters(8.0)
+            },
+            turn_on_red: name.city.country == "us" && name.city.city != "nyc",
+        },
+        onstreet_parking: match name.city.city.as_ref() {
+            "seattle" => {
+                convert_osm::OnstreetParking::Blockface(name.city.input_path("blockface.bin"))
+            }
+            "lyon" | "milwaukee" | "montreal" | "tel_aviv" | "zurich" => {
```

Comment:
This is an example where people probably copied an override around. I haven't tracked down the original place that used this.